functions/package-lock.json

```diff
@@ -1,7 +1,5621 @@
 {
```

Comment:
@harshasrikara @Stevenmqnguyen What do we think about using a v2 lockfile?

Reply:
I'm good with the change, as long as it works I'm all for staying up to date with the latest version